index.html

```diff
@@ -6,14 +6,78 @@
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="css\style.css">
```

Comment:
In this line of code your CSS is not well-linked you need to change " \ " to "/" to look like this (href="css/style.css").